### PR TITLE
CFCONFIG-21 adding commands to directly add templates

### PR DESCRIPTION
**Pull request by cybersonic**

### CODE REVIEW

File: commands/cfconfig/debugtemplates/delete.cfc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		
+		// Read existing config
+		var oConfig = CFConfigService.determineProvider( toDetails.format, toDetails.version )
+			.read( toDetails.path );
+
+		// Get the caches and remove the requested one
+		var templates = oConfig.getDebuggingTemplates() ?: {};
+		templates.delete( label );	
+
+		print.redLine(templates);
```
Comment (bdw429s) [inline commands/cfconfig/debugtemplates/delete.cfc:49]: @cybersonic Is this debugging we can remove?

**Review by bdw429s — CHANGES_REQUESTED** [review]
Looks great, just one quick question about some debugging.

**Comment by cybersonic** [comment]
Oh yeah. Great catch. I will remove!

Mark Drew

**Comment by cybersonic** [comment]
I have updated the PR

**Comment by bdw429s** [comment]
FYI, I cleaned up a couple things and pushed a new version of `commandbox-cfconfig` this morning.

**Pull request merged**